src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -224,8 +196,7 @@ public IOperationResult RunPeriodicBackup(Action<IOperationProgress> onProgress)
             }
             catch (OperationCanceledException)
             {
-                operationCanceled = TaskCancelToken.Token.IsCancellationRequested &&
-                                    _databaseShutdownCancellationToken.IsCancellationRequested;
```

Comment:
why we removed this?

Reply:
TaskCancelToken.Token is _databaseShutdownCancellationToken